Source/WebCore/bindings/scripts/CodeGeneratorJS.pm

```diff
@@ -2855,7 +2855,6 @@ sub GenerateHeader
     my $interfaceName = $interface->type->name;
     my $className = "JS$interfaceName";
     my %structureFlags = ();
-
```

Comment:
random change

Reply:
I had to make this change to work around an incremental build issue with the GTK port.